sensor-board-firmware/package.yml

```diff
@@ -0,0 +1,25 @@
+dependency-dir: ../lib
+
+depends:
+  - chibios-syscalls
+  - cmp
+  - cmp_mem_access
+  - error
+  - crc
+  - version
+  - quadramp
```

Comment:
Why do we need the quadramp module ?

Reply:
no, good catch. this is an artefact form can-io

Reply:
Copied from CAN IO firmware, can be removed